packages/bpk-component-map/src/BpkMap.js

```diff
@@ -145,30 +180,32 @@ BpkMap.propTypes = {
   }),
   center: LatLongPropType,
   children: PropTypes.node,
-  containerElement: PropTypes.node,
   greedyGestureHandling: PropTypes.bool,
-  mapElement: PropTypes.node,
   mapRef: PropTypes.func,
+  className: PropTypes.string,
   onRegionChange: PropTypes.func,
   onZoom: PropTypes.func,
   panEnabled: PropTypes.bool,
   showControls: PropTypes.bool,
+  onTilesLoaded: PropTypes.func,
   zoom: PropTypes.number,
+  mapOptionStyles: MapTypeStyle,
 };
 
 BpkMap.defaultProps = {
   bounds: null,
-  center: null,
+  center: undefined,
```

Comment:
why the default value need to change to undefined?

Reply:
I followed the default value from the [documentation](https://react-google-maps-api-docs.netlify.app/#googlemap) and it suggest that it should be `undefined` when no value is set.